phpunit.xml.dist

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.1/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.0/phpunit.xsd"
```

Comment:
Is it worth doing similar as @ https://github.com/composer/composer/pull/7863/files#diff-98c12a693941ae3c4f50872403416471R4 to avoid needing to change this again in the future?

Reply:
@weshooper Done (and doing to the others)